Homework1.py

```diff
@@ -15,28 +15,32 @@ def get_data(string: str) -> list[list[int], list[int]]:
     return final_list
 
 
-def analyze_data(data: list[list[int], list[int]], statistic: str) -> float:
+def analyze_data(data: list[list[int]], statistic: str) -> float:
     """Take list of lists and outputs statistic about data."""
     # Put all numbers into one list for average and sd
     all_data = [number for lst in data for number in lst]
     average = sum(all_data) / len(all_data)
-    if statistic == "average":
-        return round(average, 1)
-    if statistic == "standard deviation":
-        sd = ma.sqrt(sum([(i - average) ** 2 for i in all_data]) / len(all_data))
-        return round(sd, 1)
+
     # Need individual list averages and covariance for cov and corr
     avg0 = sum(data[0]) / len(data[0])
     avg1 = sum(data[1]) / len(data[1])
     covariance = sum(
         (data[0][i] - avg0) * (data[1][i] - avg1) for i in range(len(data[0]))
     ) / len(data[0])
-    if statistic == "covariance":
+
+    if statistic == "average":
+        return round(average, 1)
+    elif statistic == "standard deviation":
+        sd = ma.sqrt(sum([(i - average) ** 2 for i in all_data]) / len(all_data))
+        return round(sd, 1)
+    elif statistic == "covariance":
         return round(covariance, 1)
-    if statistic == "correlation":
+    elif statistic == "correlation":
         sd0 = ma.sqrt(sum([(i - avg0) ** 2 for i in data[0]]) / len(data[0]))
         sd1 = ma.sqrt(sum([(i - avg1) ** 2 for i in data[1]]) / len(data[1]))
         return round(covariance / (sd0 * sd1), 3)
+    else:
+        return 0
```

Comment:
Instead of returning 0 here, I think it would be better to raise an exception, e.g. `ValueError(f"Unrecognized statistic {statistic}")`.